Source/CarthageKit/GitHub.swift

```diff
@@ -265,6 +300,27 @@ private func fetchAllPages(URL: NSURL, credentials: GitHubCredentials?) -> ColdS
 			let thisData = ColdSignal.single(data)
 
 			if let HTTPResponse = response as? NSHTTPURLResponse {
+				let statusCode = HTTPResponse.statusCode
+				if statusCode > 400 && statusCode < 600 && statusCode != 404 {
```

Comment:
Why do we want to ignore 404s here? Seems like they should also cause an error with the HTTP message.

Maybe we just need to handle potentially `nil` data?